pkg/cmd/cli/cmd/loginoptions.go

```diff
@@ -337,14 +337,13 @@ func (o *LoginOptions) gatherProjectInfo() error {
 			}
 		}
 
-		if current, err := oClient.Projects().Get(namespace); err == nil {
-			o.Project = current.Name
-			fmt.Fprintf(o.Out, "Using project %q.\n", o.Project)
-		} else if !kerrors.IsNotFound(err) && !clientcmd.IsForbidden(err) {
+		current, err := oClient.Projects().Get(namespace)
+		if err != nil {
```

Comment:
Wait, you'll fail to list available projects on login if the new user doesn't have access to the previously current project, right?